Common/include/CConfig.hpp

```diff
@@ -802,7 +802,7 @@ class CConfig {
   Beta_Factor,          /*!< \brief Value of the epsilon^2 multiplier for Beta for the incompressible preconditioner. */
   Gas_Constant,         /*!< \brief Specific gas constant. */
   Gas_ConstantND,       /*!< \brief Non-dimensional specific gas constant. */
-  Molecular_Weight,     /*!< \brief Molecular weight of an incompressible ideal gas (g/mol). */
+  *Molecular_Weight,    /*!< \brief Molecular weight of an incompressible ideal gas (g/mol). */  
   Specific_Heat_Cp,           /*!< \brief Specific heat at constant pressure. */
   Specific_Heat_CpND,         /*!< \brief Non-dimensional specific heat at constant pressure. */
```

Comment:
As i mentioned, it is fine to break these bulk-declarations if it makes sense, and keeping `nMolecular_Weight` close to `*Molecular_Weight` is such a case for me. (In any case, the additional whitespaces needed to be removed here)
```suggestion
  *Molecular_Weight;    /*!< \brief Molecular weight of an incompressible ideal gas (g/mol). */
  unsigned short nMolecular_Weight; /*!< \brief Number of species molecular weights. */
  su2double Specific_Heat_Cp, /*!< \brief Specific heat at constant pressure. */
  Specific_Heat_CpND,         /*!< \brief Non-dimensional specific heat at constant pressure. */
```